src/PhpSpec/Formatter/Presenter/Exception/GenericPhpSpecExceptionPresenter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Formatter\Presenter\Exception;
+
+final class GenericPhpSpecExceptionPresenter
+    extends AbstractPhpSpecExceptionPresenter
+    implements PhpSpecExceptionPresenter
```

Comment:
There is no hard restriction on line length in PSR-2, only a recommendation to avoid making lines too long.

and all other code in phpspec keeps the class declaration on a single line IIRC (btw, the Symfony CS, which are built on top of PSR-2, are indeed mandating to keep in on a single line)